### ISSUE TITLE
Wegen Kosten zurückgezogen | Identische Anfrage stellen

### ISSUE THREAD

**Issue by Ameinsenbaer** [okfde/froide#274]
Einen Button 'Identische Anfrage in Ihrem Namen stellen', wenn eine Anfrage wegen Kosten zurückgezogen wurde. Außerdem eine Benachrichtigung an den Steller der Ursprungsanfrage, damit dieser das Ergebnis auch erfahren kann.

**Comment by arnese** [comment]
Ja, das ist gut. Sollte es auch für eingeschlafene Anfragen geben.

**Comment by re4jh** [comment]
... und für zurückgezogene Anfragen. Da laufen gerade sehr viele auf, wegen der Datenweitergabe des Anfragers bei TopfSecret.

**Comment by rugk** [comment]
evt. sollte aber (auch) explizit erklärt werden, warum das sinnvoll ist.

zB:
* Die Anfrage wurde aufgrund von Kosten zurück gezogen. Du kannst diese Kosten tragen? -> Stelle hier eine identische Anfrage.
* Die Anfrage wurde wegen potentieller Datenweitergabe zurück gezogen. Deine Daten dürfen an die betroffene Drittpartei weiter gegeben werden? -> Stelle hier eine identische Anfrage.
